.hemtt/base.toml

```diff
@@ -0,0 +1,9 @@
+name = "lambs_danger"
+prefix = "lambs_danger"
```

Comment:
Wouldn't that change the prefix for everything, so addon main is then `lambs_danger_main` and the addon danger is then `lambs_danger_danger` ?

Reply:
yes I would not change that. also the "new" Template is as far I understood beeing reworked so i would not push that currently